### DISCUSSION TITLE
哪里可以看到namesrv、broker、proxy 完整的命令行选项 以及配置项🤔️

### DISCUSSION THREAD

**Discussion by DouFuJuShi** [apache/rocketmq#7237] [Q&A]
文档上没看到关于namesrv、broker、proxy 完整的命令行选项以及配置项，有地址可以看么？

**Comment by joeCarf** [comment]
或许可以在脚本中使用 --help?

**Reply by DouFuJuShi** [reply]
不全

**Comment by wjtien** [comment]
确实，我服了，官网的也不全，也没有像类似etcd那样的带注释版的示例配置文件（namesrv, broker, proxy）。

**Comment by wjtien** [comment]
虽然这是开源项目，用爱发电类型 ，可能会被回一句爱用不用。。

**Comment by leizhiyuan** [comment]
broker https://github.com/apache/rocketmq/blob/develop/distribution/bin/mqbroker
ns https://github.com/apache/rocketmq/blob/develop/distribution/bin/mqnamesrv
proxy https://github.com/apache/rocketmq/blob/develop/distribution/bin/mqproxy
